felix/cmd/calico-bpf/commands/counters.go

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) 2022 Tigera, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package commands
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"strings"
+
+	"github.com/projectcalico/calico/felix/bpf/counters"
+
+	"github.com/olekukonko/tablewriter"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+// countersCmd represents the counters command
+var countersCmd = &cobra.Command{
+	Use:   "counters",
+	Short: "Show and reset counters",
+}
+
+func init() {
+	countersCmd.AddCommand(countersDumpCmd)
+	countersCmd.AddCommand(countersFlushCmd)
+	rootCmd.AddCommand(countersCmd)
+
+	countersDumpCmd.Flags().String("iface", "", "Interface name")
+	countersDumpCmd.Flags().Bool("all", false, "All interfaces")
```

Comment:
I think you should drop the `--all` parameter, it just makes two ways to do the same thing, which is usually a bad idea.

Reply:
Fixed.

Reply:
Fixed.